scripts/translation.js

```diff
@@ -43,13 +43,10 @@
 		if (!this.searchLang) {
 			this.searchLang = this.lang;
 		}
-		translationFile = this.rootPath + 'translations/' + this.lang + '.js';
-		this.importTranslationFile(translationFile).then(function(result) {
-			collapsedLang = thisObj.lang.replace('-','');
-			thisObj.tt = eval(collapsedLang);
-			deferred.resolve();
-		});
-		return deferred.promise();
+		translationFile = require("../translations/" + this.lang + ".js");
```

Comment:
This effectively has removed the code split point - previously only the required language was downloaded, whereas now all languages are included with the library.

This could be considered a good thing or not (less requests, but more unecessary code). If you want to preserve the split point, you can use `import()` (returns a promise) instead of `require()`

Reply:
sounds good!